### DISCUSSION TITLE
Поддержка SSR

### DISCUSSION THREAD

**Discussion by roonready** [yandex-maps-unofficial/vue-yandex-maps#228] [Q&A (2.x)] [answered]
Буду благодарен , если вы доступным языком поясните значение этой заявленной вами фичи.

**Comment by daniluk4000** [comment, marked as answer]
Пожалуйста.

1. Компонент не взаимодействует с DOM'ом на SSR. Нет необходимости делать модуль client-only
2. При отрисовке на SSR задаётся только пустой блок со стилями ширины и высоты
3. Ну и сюда же входит поддержка Nuxt

**Reply by roonready** [reply]
Спасибо. Ну то есть это не та ситуация , когда юзер может увидить в браузере с отключенным javascript  некую  первичную , хоть и не интерактивную картинку ( что тоже можно назвать поддержкой SSR в неком более жестком значении определения этой фичи) ?

**Reply by daniluk4000** [reply]
Нет, такого библиотека не умеет (там пустой див), да и не думаю, что Яндекс так может)

**Reply by roonready** [reply]
Еще раз спасибо. Выложил [минималистическую демку](https://github.com/roonready/astro-vue-yandex-maps-demo) для [Astro](https://docs.astro.build/en/guides/integrations-guide/vue/).   Возможно эту платформу тоже стоит добавить в examples

**Reply by daniluk4000** [reply]
@roonready я могу добавить в гайд по установке, конечно, а вопрос - есть какие-то особенности именно под астро, которые стоит добавлять в доку? Что было бы полезно тем, кто на нём работает, имею в виду

**Reply by roonready** [reply]
Ничего космического - некая производная от установки под vue : конфигурирование astro astro.config +  инициализация mapini.ts
но думаю такая шпаргалочка полезна. Главный мессадж - что не накстом единым ....
PS Я в стадии перевода на 3 версию api [вот этого проекта](https://www.xn--j1aci2bs.xn--d1acj3b/)

**Reply by daniluk4000** [reply]
Добавлю в следующем обновлении
